go/libraries/doltcore/diff/diff_source.go

```diff
@@ -36,14 +38,16 @@ type RowDiffSource struct {
 	joiner     *rowconv.Joiner
 	oldRowConv *rowconv.RowConverter
 	newRowConv *rowconv.RowConverter
+	sqlCtx     *sql.Context
 }
 
-func NewRowDiffSource(ad RowDiffer, joiner *rowconv.Joiner) *RowDiffSource {
+func NewRowDiffSource(ad RowDiffer, joiner *rowconv.Joiner, ctx *sql.Context) *RowDiffSource {
```

Comment:
Always want to put the ctx param (if any) first in golang methods, it's like a biblical law